D8nid 565

neilblair opened the pull request:
Theming for revision comparison.

neilblair committed:
Initial styling to fix layout when comparing revisions.

neilblair committed:
Styles for diff revision comparisons and remove general styling provided by the diff module.

neilblair committed:
Merge branch 'development' of github.com:dof-dss/nicsdru_origins_theme into d8nid-565

neilblair committed:
Typo fix

neilblair committed:
Add css for revision comparison into origin theme authenticated-styles library

neilblair committed:
Adds view mode to node__field__ theme suggestions so that we can define twig overrides for fields in the default view (which is the initial view mode for revision comparisons)

neilblair committed:
Removing diff related styles from drupal-admin.scss and placing them in their own drupal-diff.scss

neilblair committed:
Re-aligns field.html.twig with the classy theme version instead of the stable theme version.  Classy version has more drupalish css hooks and can be handy.

neilblair committed:
Ps nested in LIs (which can happen if the ckeditor normal style is applied to lists) should not have margins.

neilblair committed:
Sass compile for production

neilblair committed:
Merge branch 'development' of github.com:dof-dss/nicsdru_origins_theme into d8nid-565

David-Corbett reviewed (commented):
Couple of things to fix in the .theme file which are being flagged by phpcs, but have approved as they are fairly minor & shouldn't need to be re-reviewed

neilblair committed:
Tweak to fix diff styling for images - sometimes diff wraps an image in .diffimg and sometimes it wraps it in .diffmod.photo!

neilblair committed:
Added a fricken fullstop to a comment to satisfy the grammar and punctuation gods.
Removed an errant space preceding the concat operator lest it causes sleepless nights.
